Client/src/Utils/Report/reportStyles.css

```
@@ -0,0 +1,20 @@
+.product-report{
+    display: flex;
+    padding: 1rem;
+    color: rgb(36, 36, 36);
+    padding: 25px;
+}
+
+.report-content{
+
+}
```

Biome (1.9.4)

[error] An empty block isn't allowed.

Consider removing the empty block or adding styles inside it.

(lint/suspicious/noEmptyBlock)